tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/web/WebServerDTOConfigTest.java

```diff
@@ -0,0 +1,93 @@
+      properties.put(ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix() + "bindings.test-binding.excludedCipherSuites", "test-excludedCipherSuites,3");
+      properties.put(ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix() + "bindings.test-binding.keyStorePassword", "test-keyStorePassword");
+      properties.put(ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix() + "bindings.test-binding.trustStorePassword", "test-trustStorePassword");
+      Configuration configuration = new ConfigurationImpl();
+      configuration.parsePrefixedProperties(webServer, "web", properties, ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix());
+
+      BindingDTO testBinding = webServer.getAllBindings().stream().filter(binding -> "test-binding".equals(binding.getName())).findFirst().get();
+      Assert.assertEquals("test-uri", testBinding.getUri());
+      Assert.assertEquals(true, testBinding.getClientAuth());
+      Assert.assertEquals("test-passwordCodec", testBinding.getPasswordCodec());
+      Assert.assertEquals("test-keyStorePath", testBinding.getKeyStorePath());
+      Assert.assertEquals("test-trustStorePath", testBinding.getTrustStorePath());
+      Assert.assertEquals("test-includedTLSProtocols,0", String.join(",", testBinding.getIncludedTLSProtocols()));
+      Assert.assertEquals("test-excludedTLSProtocols,1", String.join(",", testBinding.getExcludedTLSProtocols()));
+      Assert.assertEquals("test-includedCipherSuites,2", String.join(",", testBinding.getIncludedCipherSuites()));
+      Assert.assertEquals("test-excludedCipherSuites,3", String.join(",", testBinding.getExcludedCipherSuites()));
+      Assert.assertEquals("test-keyStorePassword", testBinding.getKeyStorePassword());
+      Assert.assertEquals("test-trustStorePassword", testBinding.getTrustStorePassword());
+   }
+
+   @Test
+   public void testSetWebBindingAppProperties() throws Throwable {
+      WebServerDTO webServer = new WebServerDTO();
+      Properties properties = new Properties();
+      properties.put(ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix() + "bindings.test-binding.apps.test-app.url", "test-url");
+      properties.put(ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix() + "bindings.test-binding.apps.test-app.war", "test-war");
+      Configuration configuration = new ConfigurationImpl();
+      configuration.parsePrefixedProperties(webServer, "web", properties, ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix());
+
+      BindingDTO testBinding = webServer.getAllBindings().stream().filter(binding -> "test-binding".equals(binding.getName())).findFirst().get();
+      AppDTO testApp = testBinding.getApps().stream().filter(app -> "test-app".equals(app.getName())).findFirst().get();
+      Assert.assertEquals("test-url", testApp.getUrl());
+      Assert.assertEquals("test-war", testApp.getWar());
+   }
+}
```

Comment:
maybe one more test, that verifies feedback when an attribute is not found or misspelled. but otherwise this is great :-)

Reply:
I added other tests to improve the coverage.